docs/api/util.rst

```diff
@@ -0,0 +1,18 @@
+====================
+Utilities
+====================
+
+FileCopier
+------------
+:meth:`Flow.get <bionic.Flow.get>` can return a
```

Comment:
nit: I might add "When called with the `mode='FileCopier'` argument", or similar.

Reply:
Adding!